tests/pages/pmmSettingsPage.js

```diff
@@ -180,7 +200,7 @@ module.exports = {
     alertmanagerRulesLabel: '$alertmanager-rules-label',
     alertmanagerButton: '$alertmanager-button',
     amUrlLabel: locateLabel('form-field-am-url'),
-    applyButton: '//button[@type="submit"]',
```

Comment:
Your tests fail on Nightly exactly for the same reason, https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-test-nightly/detail/pmm2-ui-test-nightly/909/tests

Reply:
fixed, I've missed that I've changed it to css selector